src/components/CourseTable/index.jsx

```diff
@@ -67,6 +68,7 @@ class CourseTable extends React.Component {
       ...course,
       title: (<Link to={`/courses/${course.uuid}`}>{course.title}</Link>),
       owners: course.owners ? course.owners.map(owners => owners.name).join(', ') : '',
+      modified: moment.utc(course.modified).format('MMM DD, YYYY'),
```

Comment:
I'm not sure if that decision was made before this ticket was created, I'll look into that!

Reply:
I think we never came to a decision. It will temporarily be a bit useless with the management command because it will read when the command was run, but it could add value in the long term. ¯\_(ツ)_/¯